R/WorkbookClass.R

```diff
@@ -17819,7 +17819,8 @@ Workbook$methods(
 
     ## Increment priority of conditional formatting rule
     if (length((worksheets[[sheet]]$conditionalFormatting)) > 0) {
-      for (i in length(worksheets[[sheet]]$conditionalFormatting):1) {
+      for (i in order(seq_along(worksheets[[sheet]]$conditionalFormatting),
```

Comment:
I'm wondering if this is intended?

Reply:
oh nevermind. missed the `:1` while looking at the pull request this morning. I might have tried `rev()`, but it's fine this way

Reply:
changed to rev